packages/paste-core/components/form/test-utils/index.ts

```diff
@@ -0,0 +1,29 @@
+export interface AttributesMap {
+  class?: string;
+  'data-testid'?: string;
+  id?: string;
+  'aria-invalid'?: string;
+  disabled?: boolean;
+  label?: string;
+  'data-attr'?: string;
+  selected?: string;
+  title?: string;
+  spellcheck?: string;
+  hidden?: boolean;
+  draggable?: boolean;
+  accesskey?: boolean;
+  autocomplete?: boolean;
+  form?: string;
+}
+
+export const createAttributeMap = (element: HTMLElement): AttributesMap => {
+  const attributesMap = {};
+  const attributesNodeList: NamedNodeMap = element.attributes;
+  // eslint-disable-next-line unicorn/no-for-loop
```

Comment:
FYI I was getting two conflicting lint errors so I disabled this line. This lint rule required the use of for-in iterator instead of a loop, but after changing to use that iterator, another lint error came up instructing to _not_ use iterators, but instead use a loop.

Reply:
If you didn't want to do that, you could do something like

```
Object.keys(attributesNodeList)
  .map((item) => attributesNodeList[item])
  .map(attr => attributesMap[attr.name] = attr.value)
```

Reply:
Ah interesting, I like the work-around. I wonder how the TypeScript will like this... time to investigate!